File: beecrowd/1094.py
quant = int(input())
total = totsap = totcoe = totrat = 0
for c in range(0, quant):
    teste = input().split()
    numcob = int(teste[0])
    tipocob = str(teste[1]).upper()
    total += numcob
    if tipocob == 'C':
        totcoe += numcob
    elif tipocob == 'R':
        totrat += numcob
    elif tipocob == 'S':
        totsap += numcob
perccoe = totcoe * 100 / total
percrat = totrat * 100 / total
percsap = totsap * 100 / total
print(f'Total: {total} cobaias')
print(f'Total de coelhos: {totcoe}')
print(f'Total de ratos: {totrat}')
print(f'Total de sapos: {totsap}')
print(f'Percentual de coelhos: {perccoe:.2f} %')
print(f'Percentual de ratos: {percrat:.2f} %')
print(f'Percentual de sapos: {percsap:.2f} %')

File: beecrowd/1094.c
#include <stdio.h>

int main() {
    
    int testes, quantia, totCobaias, totsapos, totcoelhos, totratos;
    float percRatos, percSapos, percCoelhos;
    char animais, simbolo;

    simbolo = '%';
    totratos = totsapos = totcoelhos = totCobaias = 0;
    scanf("%d", &testes);

    while(testes > 0) {
        testes--;
        scanf("%d %c", &quantia, &animais);
        totCobaias += quantia;
        if(animais == 'C') {
            totcoelhos += quantia;
        }else if(animais == 'R') {
            totratos += quantia;
        }else 
            totsapos += quantia;
    }
    percCoelhos = ((double) totcoelhos * 100) / totCobaias;
    percRatos = ((double) totratos * 100) / totCobaias;
    percSapos = ((double)totsapos * 100) / totCobaias;

    printf("Total: %d cobaias\n", totCobaias);
    printf("Total de coelhos: %d\n", totcoelhos);
    printf("Total de ratos: %d\n", totratos);
    printf("Total de sapos: %d\n", totsapos);
    printf("Percentual de coelhos: %.2f %c\n", percCoelhos, simbolo);
    printf("Percentual de ratos: %.2f %c\n", percRatos, simbolo);
    printf("Percentual de sapos: %.2f %c\n", percSapos, simbolo);
    
    return 0;
}